build-system/common/git.js

```diff
@@ -179,13 +179,7 @@ function gitCommitterEmail() {
  * @return {!Array<{sha: string, isCherryPick: boolean}>}
  */
 function gitCherryMaster() {
-  return getStdout('git cherry master')
-    .trim()
-    .split('\n')
-    .map((line) => ({
-      isCherryPick: line.substring(0, 2) == '- ',
-      sha: line.substring(2),
-    }));
```

Comment:
Some musings: we had these lines originally to forcefully indicate non-cherry-pick commits. The real issue we have here is that it's (afaiu) programmatically impossible to tell whether a modified commit is a indeed cherry pick or not

This resolved the issue at hand, but returns the original issue that this was supposed to fix, which is that we wanted to have an indicator for when somebody is trying to release a potentially unauthorized version. Not sure how to resolve both of these issues together